sdk/src/main/scala/com/horizen/forge/ForgeMessageBuilder.scala

```diff
@@ -221,6 +222,11 @@ class ForgeMessageBuilder(mainchainSynchronizer: MainchainSynchronizer,
           .toSeq
       }
 
+    Try{nodeView.state.findIncompatibleTransaction(transactions.map(tx => tx.asInstanceOf))} match {
```

Comment:
1. Do we need to check it here? Can we rely on mempool correctness?
2. The results interpretation seems strange. So, if `nodeView.state.findIncompatibleTransaction` will find a problem tx, it should return "good" result, but it's processed in the opposite way.
We need to have different method name for `findIncompatibleTransaction`

Reply:
done